src/YTree.cc

```diff
@@ -169,3 +169,34 @@ YTree::hasMultiSelection() const
 {
     return ! YSelectionWidget::enforceSingleSelection();
 }
+
+YTreeItem *
+YTree::findItem( std::vector<std::string> & path ) const
+{
+    return findItem( path.begin(), path.end(), itemsBegin(), itemsEnd());
+}
+
```

Comment:
Coding style: Please use at least two blank lines between functions. That improves readability a lot.
Other code locations accordingly.

Reply:
Fixed.